docs/acs-private-apis.md

```diff
@@ -155,7 +155,7 @@ Get answer for a question.
     * Body (entry of):
         * `question` [string]
         * `answer` [string]
-        * `isCompleted` [boolean]
+        * `isComplete` [boolean]
```

Comment:
For me (slightly old code) this returns `complete` rather than `isComplete` or `isCompleted`.

Reply:
Ok - updated and I see this is now `private boolean isComplete;` - so great :+1: